### PR TITLE
Fix/SER-3088 : Filter sms campaigns list by country & Add env var for Keycloak Token URL

### PR DESCRIPTION
**Pull request by amydoxym**
## Description
- Filter sms campaigns list by country
- Add env var for Keycloak Token URL

## Changes Included in PR
- [SER-3088](https://oneacrefund.atlassian.net/browse/SER-3088)

## Screenshots, if any
image

.

[SER-3088]: https://oneacrefund.atlassian.net/browse/SER-3088?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by gilclefes — APPROVED** [review]
looks good to me

**Review by simiyu17 — APPROVED** [review]
Looks good

**Review by shejasabin — APPROVED** [review]

File: src/app/organization/organization.service.ts [hunk 1]
```diff
@@ -330,7 +330,9 @@ export class OrganizationService {
    * @returns {Observable<any>} SMS Campaigns data
    */
   getSmsCampaigns(): Observable<any> {
-    return this.http.get('/smscampaigns');
+    let countryId = JSON.parse(sessionStorage.getItem('selectedCountry'))?.id;
```
Comment (Yann-J) [inline src/app/organization/organization.service.ts:333]: I think maybe a small wrapper service to get/set these session variables could be useful, to provide some error handling...

**Review by Yann-J — APPROVED** [review]

File: src/app/organization/organization.service.ts (hunk 1 above)
Comment (amydoxym) [reply src/app/organization/organization.service.ts]: Good catch. Applied the suggestion

**Pull request merged**